flink-libraries/flink-cep/src/test/java/org/apache/flink/cep/operator/CEPRescalingTest.java

```diff
@@ -64,7 +68,7 @@ public Integer getKey(Event value) throws Exception {
 
 		Event startEvent1 = new Event(7, "start", 1.0);
 		SubEvent middleEvent1 = new SubEvent(7, "foo", 1.0, 10.0);
-		Event endEvent1=  new Event(7, "end", 1.0);
+		Event endEvent1 =  new Event(7, "end", 1.0);
```

Comment:
while we're at it we can remove the second space after `=`. I remember there were other occurrences of this.